### ISSUE TITLE
AMP embed block iframe

### ISSUE THREAD

**Issue by vsalvino** [coderedcorp/coderedcms#120]
`iframe` tag is disallowed by AMP and should use `amp-iframe` tag instead. See: https://www.ampproject.org/docs/reference/components/amp-iframe

Currently, embed blocks render with an iframe as so (the iframe portion comes from wagtail embeds directly):

```html
<div class="block-embed_video">
  <div class="">
    <div class="embed-responsive embed-responsive-16by9">
      <iframe width="612" height="344" src="https://www.youtube.com/embed/-n1XRzq_bns?feature=oembed" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
    </div>
  </div>
</div>
```

**Comment by corysutyak** [comment]
This is partially addressed in PR #207 .  With this PR, all iframes are rendered using the `amp-iframe` during AMP views.  

But, a lot of traditional iframes (youtube videos, tweets, etc) break and/or import invalid javascript when converted.  

To fix this we will need to dynamically change the iframe tag to whatever amp component (https://amp.dev/documentation/components/) best fits the content.  Luckily, embedly (which is used to embed these iframes) returns a `provider_name` attribute which will help us create a map for the tags.

**Comment by vsalvino** [comment]
Merged to master for 0.17
